apps/vx-scan/backend/src/util/exporter.ts

```diff
@@ -7,6 +7,7 @@ import { SCAN_ALLOWED_EXPORT_PATTERNS } from '../globals';
 export function buildExporter(): Exporter {
   const exporter = new Exporter({
     allowedExportPatterns: SCAN_ALLOWED_EXPORT_PATTERNS,
+    getUsbDrives,
```

Comment:
Open to pushback here, but I needed a way to make the exporter easier to test with. LMK if we should use a different pattern here - didn't see any existing tests that depend on `Exporter` (besides the exporter tests themselves, which are able to mock `getUsbDrives` directly